source/main.py

```
@@ -87,39 +86,48 @@ def maybe_base64_decode(s: str) -> str:
+        if proto == "shadowsocks" and not host:
+            body = link.split("ss://", 1)[1]
+            if "#" in body:
+                body, _ = body.split("#", 1)
+            try:
+                decoded = b64_decode(body)
+                if "@" in decoded:
+                    _, server = decoded.rsplit("@", 1)
+                    if ":" in server:
+                        host, port = server.rsplit(":", 1)
+            except Exception:
+                pass
```

Ruff (0.12.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)